core/src/toga/images.py

```diff
@@ -67,3 +100,35 @@ def save(self, path: str | Path):
         :param path: Path where to save the image.
         """
         self._impl.save(path)
+    
+    def as_format(self, format: Any|None=None):
+        """
+        get the image as specified format if supported
+        :param format: A supported type of Image
+        Supported types are `PIL.Image.Image`, `toga.Image`
+        :returns: Object of the specified type
+        ```
+        from PIL import Image as PIL_Image
+        pil_image = toga_image.as_format(PIL_Image.Image)
+        toga_img = toga_image.as_format(toga.Image)
+        ```
+        """
+        if format==None:
+            return self
+        elif isinstance(format, type(self)):
+            return self
```

Comment:
It's not clear to me why `as_format()` shouldn't *require* a single argument - the format you want the image in. 

It's also not clear to me why `as_format(toga.Image)` is something anyone would ever invoke... on toga.Image.